backend/dataall/base/utils/api_handler_utils.py

```diff
@@ -0,0 +1,181 @@
+import datetime
+import json
+import os
+import logging
+
+from graphql import parse, utilities, OperationType, GraphQLSyntaxError
+from dataall.base.aws.parameter_store import ParameterStoreManager
+from dataall.base.db import get_engine
+from dataall.base.services.service_provider_factory import ServiceProviderFactory
+from dataall.core.permissions.services.tenant_permissions import TENANT_ALL
+from dataall.core.permissions.services.tenant_policy_service import TenantPolicyService
+from dataall.modules.maintenance.api.enums import MaintenanceModes, MaintenanceStatus
+from dataall.modules.maintenance.services.maintenance_service import MaintenanceService
+from dataall.base.config import config
+from dataall.core.permissions.services.tenant_policy_service import TenantPolicyValidationService
+
+logger = logging.getLogger()
+logger.setLevel(os.environ.get('LOG_LEVEL', 'INFO'))
```

Comment:
What is this used for? We are using "log" everywhere else.

Reply:
Do you mean why are we using the setLevel command ?

Reply:
the setLevel method is used mostly at the top of logger initialization. I am adding it here to set the log level to whatever is set for each lambda in their environment variable - which is available via the `os.environ.get('LOG_LEVEL', 'INFO')`.